packages/plugin-connector/src/data-sources/get-sources.js

```diff
@@ -210,6 +210,19 @@ export async function cleanParquetFiles(dataDir, hashes) {
 					await fs.rm(path.join(queryPath, resultHash), { recursive: true, force: true });
 				}
 			}
+
+			if (!sourceHashes[queryName]) continue;
+			const queryHashPath = path.join(queryPath, /** @type {string} */ (sourceHashes[queryName]));
+			const timestamps = await fs.readdir(queryHashPath);
+			const latest = Math.max(...timestamps.map((x) => Number(x))).toString();
```

Comment:
This will break if run against a project that still has hashes

Reply:
TIL `Math.max(1, 2, NaN)` is `NaN`... thanks Javascript

Reply:
https://www.destroyallsoftware.com/talks/wat

Reply:
@ItsMeBrianD this was so good